### PR TITLE
feat: support custom secret provider

### PR DESCRIPTION
**Pull request by KalleV**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "@loopback/context": "^1.0.0",
     "@loopback/core": "^1.0.0",
     "@loopback/rest": "^1.1.0",
-    "express-jwt": "^5.3.1",
+    "express-jwt": "github:KalleV/express-jwt#dad0daebe354a9ad0fc8ab160e406fd5a3cac7d9",
```
Comment (keatsk) [inline package.json]: Can we avoid dependencies on your private repo?

Comment (KalleV) [reply package.json]: @keatsk I wish we could, but express-jwt refuses to merge my PR: https://github.com/auth0/express-jwt/pull/203. They're using an outdated function detection check.

File: README.md [hunk 2]
```diff
@@ -110,9 +110,19 @@ class MyController {
...
-      return 'users';
-    }
+  async users(): Promise<string> {
+    return 'users';
+  }
+
+  // This route has a dynamic scope parameter for validation.
+  // It will block a JWT that does not contain the "tenantId" value received in the route path.
+  @authenticate({
+    scope: ['{path.tenantId}:read:users']
```
Comment (dkrantsberg) [inline README.md]: is this a unique string tenant id or numeric tenant id?

Comment (KalleV) [reply README.md]: It can be whatever you want it to be.

File: package.json (hunk 1 above)
Comment (keatsk) [reply package.json]: Can we move the fork into Labshare?

Comment (KalleV) [reply package.json]: Sure, I don't have permission to do so though

**Pull request merged**